### PR TITLE
Feat/639 bam

### PR DESCRIPTION
**Pull request by cy**
~~almost gotta play with auth & error msgs~~ done

### CODE REVIEW

**Comment by cy** [comment]
this is ready for review @NCI-GDC/oicrdevs

File: modules/node_modules/@ncigdc/utils/download/index.js [hunk 1]
```diff
@@ -5,15 +5,29 @@ import _ from 'lodash';
...
 // const cookiePath = document.querySelector('base').getAttribute('href')
 const cookiePath = '/';
 const iFrameIdPrefix = '__downloader_iframe__';
 const formIdPrefix = '__downloader_form__';
 const getIframeResponse = iFrame => JSON.parse(iFrame.contents().find('body pre').text());
 const showErrorModal = error => {
   const warning = error.warning || error.message;
-
-  // open modal
+  store.dispatch(setModal(
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/utils/download/index.js]: I think this util is used across the site, so bam slicing specific stuff probably shouldn't be in it?

Comment (cy) [inline modules/node_modules/@ncigdc/utils/download/index.js]: It's a pretty generic error modal, attempts to get the message out of the error and there isn't one it displays  "GDC download service is currently experiencing issues."

Comment (Jephuff) [inline modules/node_modules/@ncigdc/utils/download/index.js]: sorry, I should have tagged a couple lines down. it says `BAM Slicing Failed`?

Comment (cy) [inline modules/node_modules/@ncigdc/utils/download/index.js]: oh yeah, guess that's not really needed. I'll remove it

**Comment by cy** [comment]
comment addressed, test added ready for review again

File: modules/node_modules/@ncigdc/components/File.js [hunk 2]
```diff
@@ -175,7 +179,7 @@ const File = ({ node, theme, searchTerm, setState }) => {
   );
 };
 
-export default compose(
+export default connect()(compose(
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/File.js]: Any reason not to be connect inside compose?

**Pull request merged**